nwb_explorer/nwb_model_interpreter/nwb_reader.py

```diff
@@ -24,7 +26,10 @@ def get_plottable_timeseries(time_series, resampling_size=None):
 
         # TODO we may need to rearrange that when dealing with spatial series: a different type of plot (3D or else)
         #  may me more adequate than what we're doing (i.e. splitting in multiple mono dimensional timeseries)
-        time_series_array = NWBReader.get_mono_dimensional_timeseries_aux(time_series.data[::1])
+        d = time_series.data[::1]
+        if len(time_series.data.shape)==3 and time_series.data.shape[1]==1 and time_series.data.shape[2]==1:
+            d = np.array([ b[0][0] for b in time_series.data[::1]  ])
```

Comment:
Would using `d = d.reshape(time_series.data.shape[0])` be better to avoid copying data?

Reply:
Good point. Fixed.